### DISCUSSION TITLE
Where can I find iCub3 motor datasheets?

### DISCUSSION THREAD

**Discussion by isorrentino** [robotology/community#490] [Q&A] [answered]
Hi all,

I was looking for the datasheet of the iCub 3 motors. I found the motor types in the official documentation (https://icub-tech-iit.github.io/documentation/icub3/icub3-joints/) but I could not find the datasheets.

In particular, I'm interested in leg motors. Here the motor types:

|         Joint         |     Motor     |
|:---------------------:|:-------------:|
| hip_pitch<br>hip_roll | MOOG_C2900580 |
|        hip_yaw        | MOOG_C2900576 |
|  knee<br>ankle_pitch  | MOOG_C2900580 |
|       ankle_roll      | MOOG_C2900575 |

**Comment by salvi-mattia** [comment, marked as answer]
hi @isorrentino 
here u can find what u r looking 4

[Moog.zip](https://github.com/robotology/community/files/6123209/Moog.zip)

**Reply by pattacini** [reply]
We might consider publishing this info somehwere.
Stay tuned.

**Reply by isorrentino** [reply]
Thanks @salvi-mattia!

**Reply by pattacini** [reply]
@isorrentino we have now online the [datasheets of some custom components][1] 🌐

[1]: https://icub-tech-iit.github.io/documentation/custom_components/custom_components

**Reply by pattacini** [reply]
cc @S-Dafarra
